index.html

```diff
@@ -0,0 +1,552 @@
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+        <tr>
+          <td>1</td>
+          <td>2450</td>
+          <td>Easy Room</td>
+          <td>Brook</td>
+          <td>Mike</td>
+          <td>10-5</td>
+          <td><button id="join-btn" type="button" class="btn btn-primary">Join +</button></td>
+        </tr>
+      </tbody>
```

Comment:
Table looks great!  For the purpose of this page being a placeholder until we actually hook it to the API, let's put only a few items in the list.  I think the important thing show are the 3 different cases we should expect ( we can just make a row for each).
- A room with 2 players and a game underway
- A room with 1 players and is available to join.
- A room with 0 players and is available to join.

Displaying what these 3 cases look like is the priority of the mock data.

Reply:
Thanks, sounds good!